dotnet/src/SemanticKernel.Abstractions/Contents/ChatMessageContent.cs

```diff
@@ -142,13 +148,64 @@ public ChatMessageContent()
     /// <summary>
     /// Creates a new instance of the <see cref="ChatMessageContent"/> class
     /// </summary>
+    /// <param name="name">Name of the author of the message</param>
+    /// <param name="role">Role of the author of the message</param>
+    /// <param name="content">Content of the message</param>
+    /// <param name="modelId">The model ID used to generate the content</param>
+    /// <param name="innerContent">Inner content object reference</param>
+    /// <param name="encoding">Encoding of the text</param>
+    /// <param name="metadata">Dictionary for any additional metadata</param>
+    public ChatMessageContent(
+        string? name,
```

Comment:
See earlier comment about how applicable this property is across multiple different LLM's.

The `metadata` property as added to allow us to add additional properties without requiring changes to the API surface.

Reply:
Ok